lib/Server.js

```diff
@@ -98,21 +98,6 @@ function getEntryModules(allModules, reactNativeExternals) {
 }
 
 function makeHotConfig(webpackConfig) {
-  // Ensure react-hot-loader is installed.
-  if (!fs.existsSync(path.resolve(process.cwd(), 'node_modules/react-hot-loader'))) {
```

Comment:
We could leave this check but I don't think it's a big deal now that we're ignorant of RHL. The "hot mode" in RNWS would just enable hot dev server now—so I think it makes sense for it to stay ignorant of babel-plugin-react-transform or react-transform-webpack-hmr.